fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -63,6 +63,7 @@
 import org.apache.iceberg.hadoop.HadoopOutputFile;
 import org.apache.iceberg.hadoop.SerializableConfiguration;
 import org.apache.iceberg.hadoop.Util;
+import org.apache.iceberg.io.ByteBufferInputStream;
```

Comment:
IcebergCachingFileIO additionally supports caching file streams to disk. you need to check this if works without starrocks code.

Reply:
Yes, i have seen that SR extended tha disk cache. So i only removed the three class `ByteBufferInputStream.java `MultiBufferInputStream.java` `SingleBufferInputStream.java`, which are total same with iceberg's codebase.

And I didn't remove `IOUtil.java` as it has beed made some custom modifications to support the disk cache.